src/TriggerCandidateMakerTrackFinding.cpp

```diff
@@ -0,0 +1,166 @@
+/**
+ * @file TriggerCandidateMakerTrackFinding.cpp
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2021.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#include "triggeralgs/TrackFinding/TriggerCandidateMakerTrackFinding.hpp"
+
+#include "TRACE/trace.h"
+#define TRACE_NAME "TriggerCandidateMakerTrackFindingPlugin"
+
+#include <vector>
+#include <math.h>
+using namespace triggeralgs;
+
+void
+TriggerCandidateMakerTrackFinding::operator()(const TriggerActivity& activity,
+                                                std::vector<TriggerCandidate>& output_tc)
+{
+
+  std::vector<TriggerActivity::TriggerActivityData> ta_list = { static_cast<TriggerActivity::TriggerActivityData>(
+    activity) };
```

Comment:
`ta_list` doesn't get used anywhere. Is this intentional?

Reply:
removed